src/serlo-editor/plugins/image/controls/settings-modal-controls.tsx

```diff
@@ -1,15 +1,10 @@
-import { faRedoAlt } from '@fortawesome/free-solid-svg-icons'
-
-import { UploadButton } from './upload-button'
 import { ImageProps } from '..'
-import { FaIcon } from '@/components/fa-icon'
 import { useEditorStrings } from '@/contexts/logged-in-data-context'
 import { tw } from '@/helper/tw'
-import { EditorTooltip } from '@/serlo-editor/editor-ui/editor-tooltip'
 import { isTempFile } from '@/serlo-editor/plugin'
 import { OverlayInput } from '@/serlo-editor/plugin/plugin-toolbar'
 
-export function SettingsModalControls({ state, config }: ImageProps) {
+export function SettingsModalControls({ state }: ImageProps) {
```

Comment:
```suggestion
export function SettingsModalControls({ state }: Pick<ImageProps, 'state'>) {
```

Minor suggestion, makes the type a bit stricter.